WebContent/WEB-INF/css/styles.css

```diff
@@ -762,4 +800,79 @@ a.reports
   87.5% {
     box-shadow: 0em -3em 0 0, 2em -2em 0 -1em, 3em 0 0 -1em, 2em 2em 0 -1em, 0 3em 0 -1em, -2em 2em 0 0, -3em 0em 0 0, -2em -2em 0 0.2em;
   }
+}
+.DannyModal{
+	position: fixed;

```

Comment:
nit: indent

Reply:
@NajibAbuSamra i do not see a problem here